packages/oca/package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@aries-framework/anoncreds": "^0.4.0",
     "@aries-framework/core": "0.4.0",
+    "axios": "^1.4.0",
```

Comment:
Could we instead rely on the Fetch API built in? Or would this be better for consistency?

Reply:
I used axios mainly because it gives me a "client" object that has a baseUrl, which is essential when I'm fetching relative paths. I tested the Fetch API and it worked fine, but axios was easier to handle relative paths

Reply:
I've also seen some implementation differences between browser Fetch API vs React Native as well.